website/pages/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+page_title: Storage
+sidebar_title: Storage
+description: >-
+  Vault relies on external storage to save its information.
+---
+
+# Storage
+
+As described on our [Architecture](/docs/internals/architecture) page, Vault's storage backend is untrusted storage used purely to keep encrypted information.
+
+## Supported Storage Backends
+
+For enterprise customers, HashiCorp provides official support for Consul and Vault's Integrated Storage as storage backends.
+
+Many other options for storage are available with community support - see our [Storage Configuration](/docs/configuration/storage) section for more information.
+
+## Backups & Restores
+
+Due to the highly flexible nature of Vault's potential storage configurations, providing exacting guidance on backing up and restoring Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
```

Comment:
What scripts are we referring here? The customer written scripts?
If yes, they will likely not be in Vault's store; and do we really want to warn about that?. It might make readers wonder about a storage related artifact to back up which doesn't exist.

Reply:
Yes - these are things that the customer might use to interact with Vault that are not stored within Vault itself. I do think it's valuable to mention both the desire to save things internal to Vault and things external to Vault as a reminder to operators for when they're coming up plans - like a DR plan, or what needs to be on a new node when it's created and added to a cluster.

So, I want to include both items, but if you have suggestions for how to rephrase the second one to better represent "external-to-Vault stuff that you as an operator might want to save", I'm happy to rephrase?

Reply:
I can see both points of view here.  Maybe some like this for 2

"Disk backup for the underlying server running Vault.  _Note: This would include Vault's configuration file and any custom scripts written to help with the management of Vault and/or the server itself_"

Reply:
Do we also want to mention here, or somewhere below, that the backups will only be able to be decrypted by the seal that is in use at the time of backup?